exercises/pangram/src/main/java/Pangrams.java

```diff
@@ -0,0 +1,6 @@
+public class Pangrams {
```

Comment:
It now appears that the object we're dealing with is no longer a "pangram"... it's a ... "sentence"?  What would the API look like if we instantiated a sentence and interrogated it as to whether it is a pangram?